### PR TITLE
Softened the color a bit by mixing in the panel-bg.

### PR DESCRIPTION
**Pull request by mattttt**
Softened the color a bit by mixing in the panel-bg, so it's not introducing any new variables. Was too stark as pure body-bg. 

I think it sits much nicer in the dashboard now when there are many info items. We want to encourage folks to enter these details, and I dont want to visually punish them if every panel had info. 

screenshot 2016-12-16 12 13 35

The light theme is fine, on-par with the rest of the light theme. But that really needs some love and contrast overall. 

Added a min-height for modal-body container. It was too small and awkward when the message was only a line or two.

### CODE REVIEW

File: public/sass/pages/_dashboard.scss [hunk 1]
```diff
@@ -102,22 +102,22 @@ div.flot-text {
...
+    top: -4px;
     left: -5px;
-    font-size: 90%;
+    font-size: 75%;
   }
 
   &--info {
     display: block;
-    background: $blue-dark;
+    background: mix($panel-bg, $body-bg, 35%);
```
Comment (torkelo) [inline public/sass/pages/_dashboard.scss]: not sure if it's a good idea mixing colors, basically sneaking in a another color, is there no color we can use from the palette?

Comment (mattttt) [inline public/sass/pages/_dashboard.scss]: I went through them all - Im comfortable mixing two existing colors in this case, but not introducing another variable. 

The pure $body-bg was too stark of a contrast, and seemed strange next to the border, and everything else is lighter. Nothing else in the darker spectrum looked good. :/

I understand the concern though - and you're objectively right. It is a new color, but it's derived from our palette, which feels like an important distinction.

**Comment by mattttt** [comment]
New changes, should be ready to roll.

**Comment by mattttt** [comment]
I should wait for you to merge this, right?

**Pull request merged**